plugin.py

```diff
@@ -25,7 +27,7 @@ def plugin_info(self):
             # the matcher specifies in HQL-Lite which traces will be processed by this plugin
             #  e.g. $data.fileType=AppleDouble
             # see also https://netherlandsforensicinstitute.github.io/hansken-extraction-plugin-sdk-documentation/latest/dev/concepts/hql_lite.html#how-to-write-a-matcher
-            matcher='$data.fileType={FILETYPE_PROPERTY}',
+            matcher='$data.mimeClass=picture',
```

Comment:
I think this change should be reverted, it seems specific to a test?

Reply:
True, will be reverted as soon as it is no longer necessary.

Reply:
done